### PR TITLE
LPS-169694 [Raylife AP]  Create graph structure

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-169694](https://issues.liferay.com/browse/LPS-169694)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.dashboard-commission-container {
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css:1]:
```suggestion
.claims-total-settlements-container {
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	top: 50px;
+}
+
+.dashboard-commission-container .dashboard-commission-select {
+	border-color: var(--white);
+	padding: 5px;
+	width: 105px;
+}
+
+.dashboard-commission-container .dashboard-commission-title::after {
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css:23]: same here 
```suggestion
.claims-total-settlements-container
	.claims-total-settlements-header
	.claims-total-settlements-title::after {
	background-color: #4d85ff;
	content: '';
	height: 1px;
	left: 28px;
	min-width: 20px;
	position: absolute;
	top: 50px;
}

.claims-total-settlements-container .claims-total-settlements-select {
	border-color: var(--white);
	padding: 5px;
	width: 105px;
}

.claims-total-settlements-container .claims-total-settlements-title::after {
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		flex-direction: row !important;
+		justify-content: space-evenly !important;
+		width: 100%;
+	}
+
+	.dashboard-commission-screen {
+		padding-left: 24px;
+		padding-right: 24px;
+	}
+}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css:54]: same here

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+export default function () {
+	const [selectedFilterDate, setSelectedFilterDate] = useState('1');
+	const claimValue = 2000;
+	const claimPercentual = 50;
+
+	const buildLegend = () => (
+		<div className="d-flex dashboard-commission-legend flex-column h-100 justify-content-end mt-5">
+			<div className="font-weight-normal mb-2 text-neutral-8 text-paragraph-sm"></div>
+
+			<div className="dashboard-commission-screen font-weight-bolder h5 mb-3">
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js:31]:
```suggestion
		<div className="d-flex claims-total-settlements-legend flex-column h-100 justify-content-end mt-5">
			<div className="font-weight-normal mb-2 text-neutral-8 text-paragraph-sm"></div>

			<div className="claims-total-settlements-screen font-weight-bolder h5 mb-3">
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+	return (
+		<div className="d-flex dashboard-commission-container flex-column flex-shrink-0 pb-4 pt-3 px-3">
+			<div className="align-items-center d-flex dashboard-commission-header justify-content-between">
+				<div className="dashboard-commission-title font-weight-bolder h4 mb-0">
+					Total Settlements Paid
+				</div>
+
+				<ClaySelect
+					className="dashboard-commission-select"
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js:98]: Same here

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+				</ClaySelect>
+			</div>
+
+			{!!chartData.columns.length && (
+				<LineChart
+					LegendElement={buildLegend}
+					chartData={chartData}
+					patternColor={['#4c84ff']}
+				/>
+			)}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js:121]:
```suggestion
                        const patternColor=['#4c84ff']
                        
                        
			{!!chartData.columns.length && (
				<LineChart
					LegendElement={buildLegend}
					chartData={chartData}
					patternColor
				/>
			)}
```

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js (hunk 6 above)
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.js:121]: Done.
Thx!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css (hunk 2 above)
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-total-settlements-paid/index.css:23]: Thx!

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @eduardo-diniz :white_check_mark:

**Comment by eduardo-diniz** [comment]
ci:test:relevant

**Comment by eduardo-diniz** [comment]
ci:test:relevant

**Review by eliasantosf — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**